core/src/main/java/com/linecorp/armeria/client/ClientBuilder.java

```diff
@@ -21,7 +21,9 @@
 
 /**
  * Creates a new client that connects to the specified {@link URI} using the builder pattern. Use the factory
- * methods in {@link Clients} if you do not have many options to override.
+ * methods in {@link Clients} if you do not have many options to override. If you want to make an
+ * {@link HttpClient}, using the {@link HttpClientBuilder} or factory methods in {@link HttpClient}
+ * is recommended.
```

Comment:
How about: `If you are creating an {@link HttpClient}, it is recommended to use the factory methods in. ..`